Gemfile

```
@@ -71,3 +71,5 @@ group :test do
   gem "selenium-webdriver"
 
 end
+
+gem "devise", "~> 4.9"
```

Add a small comment about why we are using this gem like
```suggestion
gem "devise", "~> 4.9" # User Authorization
```
or something else. Also move this gem above all the groups in file